setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setuptools.setup(
+    name="agpypeline",
+    version="0.0.1",
+    author="Jacob van der Leeuw",
+    author_email="jvanderleeuw@email.arizona.edu",
+    description="Installable package for entrypoint and environment transformers",
```

Comment:
There are no entrypoint or environment transformers. Those are components that make up a transformer. Look at the image at https://agpipeline.github.io/transformers/transformers. Please change this description and the long_description